src/css/imports/dock.less

```diff
@@ -1,12 +1,18 @@
 @import "vars";
 
 .jw-dock {
-    margin: @ui-margin/2;
+    clear: right; // This allows it to be positioned beneath the logo
+    margin: @ui-margin / 2;
     display: block;
     opacity: 1;
 
-    // This allows it to be positioned beneath the logo
-    clear : right;
+    .jw-breakpoint-0 & {
+      margin: 4px 2px;
+    }
+
+    .jw-breakpoint-1 & {
+      margin: 8px 4px;
```

Comment:
Are these making the margin smaller? It's not clear what the value of @ui-margin is. What if it's 0 0 ?